pkg/cloudprovider/provider/kubevirt/provider.go

```diff
@@ -958,3 +977,21 @@ func getTopologySpreadConstraints(config *Config, matchLabels map[string]string)
 		},
 	}
 }
+
+//func getStorageTopologies(ctx context.Context, storageClasName string, client client.Client) (map[string]string, error) {
```

Comment:
The fully commented code can be removed.

Reply:
Damn! forgot about it :D

Reply:
Done